src/github.rs

```diff
@@ -1783,6 +1781,56 @@ impl<'q> IssuesQuery for Query<'q> {
     }
 }
 
+/// Return open concerns filed in an issue under MCP/RFC process
+/// Concerns are marked by `@rfcbot concern` and `@rfcbot resolve`
+fn find_open_concerns(comments: Vec<Comment>) -> Option<Vec<(String, String)>> {
+    let re_concern_raise =
+        Regex::new(r"@rfcbot concern (?P<concern_title>.*)").expect("Invalid regexp");
+    let re_concern_solve =
+        Regex::new(r"@rfcbot resolve (?P<concern_title>.*)").expect("Invalid regexp");
+    let mut raised: HashMap<String, String> = HashMap::new();
+    let mut solved: HashMap<String, String> = HashMap::new();
+
+    for comment in comments {
+        // Parse the comment and look for text markers to raise or resolve concerns
+        let comment_lines = comment.body.split("\r\n");
```

Comment:
```suggestion
        let comment_lines = comment.body.lines();
```

Just to avoid requiring \r, which doesn't feel necessarily guaranteed?